src/main/java/com/refinedmods/refinedstorage/apiimpl/network/grid/handler/ItemGridHandler.java

```diff
@@ -31,7 +32,14 @@ public ItemGridHandler(INetwork network) {
     }
 
     @Override
-    public void onExtract(ServerPlayerEntity player, UUID id, int flags) {
+    public void onExtract(ServerPlayerEntity player, ItemStack stack, int preferredSlot, int flags) {
+        if (network.getItemStorageCache().getList().getEntry(stack, IComparer.COMPARE_NBT) != null) {
```

Comment:
Assign `network.getItemStorageCache().getList().getEntry(stack, IComparer.COMPARE_NBT)` to a variable